### DISCUSSION TITLE
Intermittent Error: Bad Request - Failed to process audio: corrupt or unsupported data

### DISCUSSION THREAD

**Discussion by taro-0** [deepgram/community#693] [General help]
### Which Deepgram product feature are you using?

Deepgram API - STT Pre-recorded

### Details

We are currently creating an integration with Deepgram using the Javascript SDK. We've had an intermittent issue with the deepgram API answering the following.
![image](https://github.com/deepgram/community/assets/5248306/5b489d25-9ae2-40bf-a3b8-d487ba955ad5)

At first, we thought it was due to the permissions in our S3 bucket. However, after testing with the sample audio file provided in your documentation, we are observing the same issue. This seems to be happening only with audio from a URL source, testing audios from a local source seem to work fine.

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

 https://api.deepgram.com/v1/listen?utterances=true&smart_format=true

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

71228a23-dd36-434c-abba-9b7411898d14

### If possible, please attach your code or paste it into the text box.

![image](https://github.com/deepgram/community/assets/5248306/8cbaad5a-c8f5-4233-abf2-f82b2d4a337d)

### If possible, please attach an example audio file to reproduce the issue.

We used the spacewalk audio demo

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
